pkg/storage/mysql.go

```diff
@@ -227,6 +235,13 @@ func (s *MysqlDB) GetStampAndJobs(hostInfo string) (int64, []string, error) {
 		return -1, nil, xerror.Wrapf(err, xerror.DB, "mysql: begin IMMEDIATE transaction failed.")
 	}
 
+	defer func() {
+		if err != nil {
```

Comment:
The `err` is a shadow variable here.

A better way is to make err as a named return value, eg:

```
func (s *MysqlDB) GetStampAndJobs(hostInfo string) (int64, []string, err error) {
```

and change the usage of err from `, err :=` to `, err = `.

Reply:
done